src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/JsonTest/JsonTest.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Data;
+using Microsoft.Data;
+using System.Data.Common;
+using System.Data.SqlTypes;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests
+{
+    public class JsonTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public JsonTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        private static readonly string jsonDataString = "[{\"name\":\"Dave\",\"skills\":[\"Python\"]},{\"name\":\"Ron\",\"surname\":\"Peter\"}]";
+
+        private void ValidateRowsAffected(int rowsAffected)
+        {
+            _output.WriteLine($"Rows affected: {rowsAffected}");
+            Assert.Equal(1, rowsAffected);
+        }
+
+        private void ValidateRows(SqlDataReader reader)
+        {
+            while (reader.Read())
+            {
+                string jsonData = reader.GetString(0);
+                _output.WriteLine(jsonData);
+                Assert.Equal(jsonDataString, jsonData);
+            }
+        }
+
+        private async Task ValidateRowsAsync(SqlDataReader reader)
+        {
+            while (await reader.ReadAsync())
+            {
+                string jsonData = reader.GetString(0);
+                _output.WriteLine(jsonData);
+                Assert.Equal(jsonDataString, jsonData);
+            }
+        }
+
+        private void ValidateSchema(SqlDataReader reader)
+        {
+            System.Collections.ObjectModel.ReadOnlyCollection<DbColumn> schema = reader.GetColumnSchema();
+            foreach (DbColumn column in schema)
+            {
+                _output.WriteLine("Column Name is " + column.ColumnName);
+                _output.WriteLine("Column DataType is " + column?.DataType.ToString());
+                _output.WriteLine("Column DataTypeName is " + column.DataTypeName);
+                Assert.Equal("json", column.DataTypeName);
+            }
+        }
+
+        [ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.IsJsonSupported))]
+        public void TestJsonWrite()
+        {       
+            string tableName = DataTestUtility.GetUniqueNameForSqlServer("Json_Test");
```

Comment:
You can consider calling DataTestUtility.DropTable() before creating table with your own name.

Reply:
`DataTestUtility.GetUniqueNameForSqlServer` ensures a unique name for the table. We can modify it to use `DataTestUtility.CreateTable()` once the streaming PR is checked into main [https://github.com/dotnet/SqlClient/pull/2801](url)